docs: Canvas: Update docs for v10 (#67793)
grafana/grafana: docs/sources/panels-visualizations/visualizations/canvas/index.md

{{< video-embed src="/media/docs/grafana/canvas-connections-9-4-0.mp4" max-width="750px" caption="Canvas connections demo" >}}

> Connections support setting both their size and color based on fixed or field values. To do so, enter into panel edit mode, select the connection, and modify the connection's properties in the panel editor.

Reviewer: This phrasing is a bit awkward; suggest:
```suggestion
You can sett both the size and color of connections based on fixed or field values. To do so, enter into panel edit mode, select the connection, and modify the connection's properties in the panel editor.
```
